### ISSUE TITLE
MySQL does not start with overlay2 on docker for OSX 

### ISSUE THREAD

**Issue by ProfessorDeGraw** [dixudx/keystone-docker#1]
On docker for OSX sometimes there is a problem with the SQL server failing to start.
I look like it is because the OverlayFS only implements a subset of the POSIX standards.

A workaround is to touch the mysql files before starting mysql

**find /var/lib/mysql -type f -exec touch {} \; && service mysql start**

[https://github.com/docker/for-linux/issues/72](https://github.com/docker/for-linux/issues/72)

`# cat /var/log/mysql/error.log`
_[ERROR] Fatal error: Can't open and lock privilege tables: Got error 140 from storage engine_

**Comment by dixudx** [comment]
@ProfessorDeGraw Thanks for reporting this. Really appreciate.

Would you mind fixing this with a new PR? The credit should go to you.
